packages/web-components/src/components/ic-search-bar/ic-search-bar.tsx

```diff
@@ -38,13 +38,17 @@ import searchIcon from "../../assets/search-icon.svg";
 
 let inputIds = 0;
 
+const LOADING_LABEL = "Loading...";
+const LOADING_ERROR_LABEL = "Loading Error";
```

Comment:
Should these two be a prop so they can be changed but have the default as they are above.